test/test_genome.py

```diff
@@ -766,3 +775,16 @@ def test_ncolumns_zero(rng):
 
     CartesianGraph(genome).to_func()
     CartesianGraph(genome).to_numpy()
+
+
+def test_slice_dna(genome_params, rng):
+
+    genome = cgp.Genome(**genome_params)
```

Comment:
just to make sure to not get false positives you could do this 10 times

Reply:
I wasn't fully sure what to do (the logic changed since you wrote that message) - but I have added a few more (complicated) expressions for the output (in `test_set_expression_for_output`)